### ISSUE TITLE
5G Broadcast Operation in MFN

### ISSUE THREAD

**Issue by haudiobe** [5G-MAG/Standards#58]
# Problem description
The UHF spectrum in most European countries, including the UK is arranged on the bases of multi-frequency networks (MFN), each transmitting station uses a different frequency, rather than single-frequency networks (SFN).  If we were to re-purpose an existing network for 5G Broadcast the network would be MFN – so a mobile receiver would need to change frequency to maintain a service. The question has been asked how does 5G Broadcast cope with operating in an MFN when the receiver is mobile – can the receiver switch between stations on different frequencies? 

# Suggested solution
5G Broadcast introduces mechanisms to enable service continuity across multiple frequencies (in standalone mode, i.e., without the need for any other connection). In particular, SIB15 includes the information about MBMS Service Area Identities (SAI) of neighboring frequencies:

_`SystemInformationBlockType15`: The IE `SystemInformationBlockType15` contains the MBMS Service Area Identities (SAI) of the current and/ or neighbouring carrier frequencies._

It is proposed to add such text into the ETSI TS 103 720

# Additional context
It would be good to add this text before v1.2.1 is published

**Comment by jordijoangimenez** [comment]
When reading TS 36.331, there are the following parameters: mbms-SAI-InterFreqList; mbms-SAI-IntraFreq; mbms-SAI-List; multiBandInfoList.

It is unclear what an MBMS SAI refers to. Is the MBMS SAI uniquely identifying a Service (TMGI)?

If I have an MBMS carrier with several services and I'm listening to service "Radio1" is transmitter area A, I need to be able to find the same "Radio1" in transmitter area B. Is there a way to attach a frequency (EARFCN) directly to a service? This is done in DAB and in FM. Case b is more a radioDNS service.

There are two subcases:
a) When moving to another transmitter area, we move to another editorial region. "Radio1-North" is going to be switched to "Radio1-South" when the vehicle crosses from region North to region South. Radio1-North and Radio1-South are independent services (independent TMGIs) although the programme/station is the same.
b) When moving to another transmitter area, there's no longer a regional service offered. Or simply there no service at all anymore offered on 5G Broadcast. The "neighbour frequency" should no longer be a frequency but just an internet service of that original radio station (which is active all time non geo-blocked).

This functionality is independent of SFN or MFN as SFN regions containing multiple TX also need to be reused in MFNs of SFN regions.

I would just treat anything as a frequency-reuse-1 MFN (which is effectively an SFN but asynchronous).

**Comment by haudiobe** [comment]
Another set of information:

In 4G/5G systems there are two types of mobility.

In IDLE mode (when the UE is not having an active unicast session with the base station), the mobility is driven by the UE. Essentially the UE makes measurements in the downlink, and switches to a different cell when the serving cell quality is below a threshold. There is information of neighbor frequencies that the UE may try to search on. 5GB mobility is based on this mode (the UE is always in IDLE). 

In connected mode,the base station requests measurements from the UE and instructs the UE to move to a different cell (this is called a “handover”). This requires uplink and downlink signaling exchange between the UE and the base station.

**Comment by haudiobe** [comment]
CR Uploaded https://member.5g-mag.com/wg/CD-T/document/1011?downloadRevision=1736

**Comment by jordijoangimenez** [comment]
I've got a proposal for some general intro:

Service continuity information is provided to the 5G Broadcast Receiver through a combination of User Service Description (USD) and System Information Block Type 15 (SIB15).
- In the USD, each 5G Broadcast Service is associated with a Temporary Mobile Group Identity (TMGI), frequency and MBMS Service Area Identity.
- In the SIB15, a list of current and neighboring frequencies together with the SAIs supported
by each frequency is provided.

To respond myself and thinking more on an implementation guideline type of comment, I would suggest that each service/TMGI is always carrying its own SAI (this gives full granularity and unique mapping... I believe like @rjb1000 mentioned about FM/RDS). Alternatively, you can map the carrier (all services within the carrer) to an SAI)... I would see this as how DVB and DAB work, in particular when there are no regional programmes and you just want to apply a frequency reuse to cover a large area. 

What happens if no neighbour frequency is specificed? I would recommend that you fall into unicast or, in absence of unicast.... you will just go into another service randomly. In FM, when you cannot decode RDS you just stay in the same frequency and either you listen nothing or you listen to another station that has nothing to do, For DAB I have no idea.

**Comment by jordijoangimenez** [comment]
To be checked if we are happy with this addition: https://member.5g-mag.com/wg/CD-T/document/1013

**Comment by jordijoangimenez** [comment]
Some additional comments. @johannmika 

Most likely, given pathloss, it is not possible to reach coverage continuity between two stations when the inter-site distance is large and power not large enough to compensate pathloss in smartphone-mobile conditions. In the path between the two stations this would translate into two scenarios:
1. BC_COVERAGE_TX1 - NOSIGNAL - BC_COVERAGE_TX2. It would be useful if the list of transmitters indicated from TX1 is stored somewhere so that the UE knows where to look for the service even when in a region without coverage.
2. BC_COVERAGE_TX1 - UNICAST - BC_COVERAGE_TX2. In the case there is connectivity and the service provided with unicast, it would be convenient if while in unicast there would still be access to a list of frequencies where to find the service so when the user reaches TX2, it is possible to jump to TX2.
3. UNICAST - BC_COVERAGE. In general, it would be useful that when the journey is started in unicast, there is a possibility to know exactly where to find this service if available in broadcast at a certain point.

**Comment by rguaus** [comment]
In case there is overlapped coverage from two transmitters in Region 1 and Region 2 using different frequencies for the same service, and in case at some times the content of the service in Region 1 is different from the content in Region 2 (ad insertion, regional news...), it might be interesting not to switch from one region to another, even though the signal is better in the other region. Some FM receivers have this function (Local vs DX), which is useful to avoid the flip-flap switching for users moving all the time around border zones.

**Comment by jordijoangimenez** [comment]
I agree and, even more, I think the whole service continuity needs to be managed at a logical level with an strict following between TMGIs and USDs.
In the old radio world you don't want your radio station to be confused by another one just because the same frequency is used in the proximity.
In the present world of apps this is even worse as you do not really want an app meant to play one radio station to be playing a radio station of a competitor...

**Comment by jordijoangimenez** [comment]
After the comments from @rjb1000
This one has editorial improvements: https://member.5g-mag.com/wg/CD-T/document/1013

However this one has substantial comments that require a new review by @haudiobe: https://member.5g-mag.com/wg/CD-T/document/1011

I started on Friday v20 of the TS initially incorporating the first contribution into 12.3 without the changes in changes. Once we get the ok from Thomas I’ll take care of all remaining changes.

**Comment by jordijoangimenez** [comment]
Merged into https://member.5g-mag.com/wg/CD-T/document/779
